x/incentives/types/msgs.go

```diff
@@ -5,16 +5,24 @@ import (
 	"time"
 
 	lockuptypes "github.com/osmosis-labs/osmosis/v10/x/lockup/types"
+	txfeestypes "github.com/osmosis-labs/osmosis/v10/x/txfees/types"
 
 	sdk "github.com/cosmos/cosmos-sdk/types"
 )
 
 const (
 	TypeMsgCreateGauge = "create_gauge"
 	TypeMsgAddToGauge  = "add_to_gauge"
+
+	createGaugeMinBaseFee = 50 * 1_000_000
+	addToGaugeMinBaseFee  = 25 * 1_000_000
```

Comment:
how were these values decided?

Reply:
Do you feel they are too high or too low?